charts/rancher-cis-benchmark/0.2.1/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "0.2.0"
+description: |
+  Run CIS benchmark tests
+name: rancher-cis-benchmark
+version: 0.2.0
```

Comment:
both appVersion and version should be 0.2.1

Reply:
Thanks, done.